install/configure_kernel_ima_module/test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+[ -z "$IMA_STATE" ] && IMA_STATE="on"
+[ -z "$IMA_POLICY" ] && IMA_POLICY="tcb"
+
+COOKIE=/var/tmp/configure-kernel-ima-module-rebooted
+TESTFILE=/var/tmp/configure-kernel-ima-module-test$$
+
+rlJournalStart
+
+  if [ ! -e $COOKIE ]; then
+    rlPhaseStartSetup "pre-reboot phase"
+        rlRun "grubby --info DEFAULT | grep '^args'"
+        rlRun "grubby --update-kernel DEFAULT --args 'ima=$IMA_STATE ima_policy=$IMA_POLICY'"
```

Comment:
Shouln't we boot it with `ima_appraise=fix` first?

Reply:
I think this is used for local attestation and we do not need it for keylime. Anyway, we can add it later if needed.

Reply:
ima_appraise=enforce is the default value so I'm not sure what is the real impact of it.